keanu-project/src/main/java/io/improbable/keanu/network/NetworkReader.java

```diff
@@ -0,0 +1,14 @@
+package io.improbable.keanu.network;
+
+import io.improbable.keanu.vertices.Vertex;
+import io.improbable.keanu.vertices.bool.BoolVertex;
+import io.improbable.keanu.vertices.dbl.DoubleVertex;
+import io.improbable.keanu.vertices.intgr.IntegerVertex;
+
+public interface NetworkReader {
```

Comment:
Given the interface methods, should it be NetworkLoader? Alternatively, the methods could be `readValue(...)`

Reply:
As below